app/views/map/_leaflet.html.erb

```diff
@@ -5,10 +5,21 @@
 <div class="leaflet-map" id="map<%= unique_id %>"></div>
 <% if defined? people %><p><i><small>Share your own location on <a href='/profile'>your profile</a>.</small></i></p><% end %>
 <script>
-  var map = L.map('map<%= unique_id %>').setView([<%= lat %>,<%= lon %>], <%= lat.to_s.length.to_i %> + 6);
+  var map = L.map('map<%= unique_id %>').on('load', onMapLoad).setView([<%= lat %>,<%= lon %>], <%= lat.to_s.length.to_i %> + 6);
   L.tileLayer("//a.tiles.mapbox.com/v3/jywarren.map-lmrwb2em/{z}/{x}/{y}.png").addTo(map);
+  function onMapLoad(e){
+	 var lat = <%= lat %> ;
+	 var lon = <%= lon %> ;
+	 var s = lat+","+lon ;
+ 	$.getJSON("https:"+"//publiclab.org/api/srch/locations?srchString="+s , function(data){
```

Comment:
Should this aim at `/` rather than `//publiclab.org/api/...`?

This is looking FANTASTIC

Reply:
Hi , thank you so much ! I think both ways are correct , what do you think ?

<img width="743" alt="screen shot 2018-01-25 at 10 53 55 pm" src="https://user-images.githubusercontent.com/14952645/35402464-adb11bf4-0222-11e8-8c0c-cb260b91f6f0.png">

Reply:
ah, i mean that if you run this codebase somewhere besides publiclab.org, we'd want to just aim the request at the root. Make sense?